README.md

```diff
@@ -91,3 +91,32 @@ test framework.
 
 If you would like to develop and update the go binaries you will need to install 
 [go 1.12](https://golang.org/doc/install#install) and [upx](https://upx.github.io/)
+
```

Comment:
Having this in the README is good, but this should probably be a DevCenter article